css/style.css

```diff
@@ -120,14 +127,72 @@ nav > ul {
   z-index: 0;
 }
 
+.social_media > li {
+  margin: 10px;
+  list-style: none;
+}
+
+.lang > li {
+  height: inherit;
+  display: grid;
+  background-color: #ebebff;
+  color: #3d51ff;
+  border-radius: 8px;
+  align-items: center;
+  text-align: center;
+  list-style: none;
+}
+
+.popupbody .lang > li {
+  font-size: 12px;
+  font-weight: 500;
+  line-height: 16px;
+}
+
 .popup {
```

Comment:
- [ ] Kindly check your styling. As you can see your **Languages content** stll overlapping the popup when you scroll
![Screenshot from 2022-02-04 23-58-08](https://user-images.githubusercontent.com/65068771/152615933-2004a060-8a0a-4014-9671-d2dd67511ff8.png)